apps/console.py

```diff
@@ -355,24 +356,30 @@ def show_services(self, services):
         lines = []
         del self.known_attributes[:]
         for service in services:
-            lines.append(('ansicyan', str(service) + '\n'))
+            lines.append(('ansicyan', f'{service}\n'))
 
             for characteristic in service.characteristics:
-                lines.append(('ansimagenta', '  ' + str(characteristic) + '\n'))
+                lines.append(('ansimagenta', '  {characteristic}\n'))
                 self.known_attributes.append(f'{service.uuid.to_hex_str()}.{characteristic.uuid.to_hex_str()}')
                 self.known_attributes.append(f'*.{characteristic.uuid.to_hex_str()}')
                 self.known_attributes.append(f'#{characteristic.handle:X}')
                 for descriptor in characteristic.descriptors:
-                    lines.append(('ansigreen', '    ' + str(descriptor) + '\n'))
+                    lines.append(('ansigreen', f'    {descriptor}\n'))
 
         self.services_text.text = lines
         self.ui.invalidate()
 
     def show_attributes(self, attributes):
         lines = []
-
         for attribute in attributes:
-            lines.append(('ansicyan', f'{attribute}\n'))
+            if type(attribute) is Service:
+                lines.append(('ansicyan', f'{attribute}\n'))
+            elif type(attribute) in (Characteristic, CharacteristicDeclaration):
+                lines.append(('ansimagenta', f'  {attribute}\n'))
+            elif type(attribute) is Descriptor:
+                lines.append(('ansigreen', f'    {attribute}\n'))
+            else:
+                lines.append(('ansiyellow', f"{attribute}"))
```

Comment:
nit: mixed single and double quotes in this section

Reply:
applied black to this section